src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -503,4 +503,16 @@
   <data name="EnableAnalyzerReleaseTrackingRuleDescription" xml:space="preserve">
     <value>Enabling release tracking for analyzer packages helps in tracking and documenting the analyzer diagnostics that ship and/or change with each analyzer release. See details at https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</value>
   </data>
+  <data name="PreferIsKindDescription" xml:space="preserve">
+    <value>Prefer IsKind</value>
```

Comment:
Can the description be improved to state why `IsKind` is preferred? What is the performance drawback from using `Kind()`

Reply:
Reworded the description to include much more detail.